pkg/KV/results.go

```diff
@@ -0,0 +1,82 @@
+package KV
+
+import "sync"
+
+type KVD struct {
+	sync.RWMutex
+	M map[string]map[int]struct{}
```

Comment:
If i understand correctly, we are using this map here to track IP Address -> Ports mapping. If yes, I'd suggest let's not use this correlation mechanism and go fully asynchronous. Such type of tracking usually becomes the bottleneck for speed. I'd say let's keep it upto the user to track results. We should only simple dump IP -> Port information in various formats on stdout like masscan and let the user do the parsing.

Reply:
Marking as solved as currently under discussion